spec/support/factory_girl.rb

```diff
@@ -17,7 +17,8 @@
   factory :image do
     user
     gallery
-    submission "mermaid.png"
+    submission { Rack::Test::UploadedFile.new(File.join(Rails.root, 'spec',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.